util/statvar_dcid_generator.py

```diff
@@ -228,6 +228,16 @@
     }
 }
 
+# This is a list of boolean properties
+_BOOLEAN_PROPS = [
```

Comment:
Are there any properties that being with words like 'is, 'has' that are either not boolean or have a different way of naming? 

If not, may be better to check if the first word (separated by camel case) is one of these and treat like a boolean property.

Reply:
looks like there are propeorties like 'hsaChildren' that are not boolean.

Would a combination of prefix and value in {true, false} siffice?

Reply:
There are properties which are not prefixed by an 'is' or an 'has' like `acceptsReservations` and `smokingAllowed`. However, I haven't found any naming conventions for these properties.

The `_BOOLEAN_PROPS` list contains all `is` and `has` properties from https://datacommons.org/browser/Boolean

Reply:
> Would a combination of prefix and value in {true, false} suffice?

Sure. However, this won't match with existing dcids like https://datacommons.org/browser/Count_Household_NoComputer

Reply:
Ok, we can revisit generalizing this later. Pls go ahead with this set.